.github/dependabot.yml

```
@@ -25,3 +25,7 @@ updates:
     labels:
       - "maintenance"
       - "dependencies"
+  - package-ecosystem: "devcontainers"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Although the PR using dependabot is no longer used for release notes nowadays, I would like to continue labeling them.
```suggestion
      interval: "monthly"
    labels:
      - "maintenance"
      - "dependencies"
```